keps/sig-storage/1710-selinux-relabeling/kep.yaml

```diff
@@ -23,12 +23,16 @@ latest-milestone: "v1.27"
 milestone:
   alpha: "v1.24"
   beta: "v1.27"
```

Comment:
Can you add milestones of both feature gates?  Can you add notes next to the "alpha" and "beta" feature gates so we know which feature gate it is associated with?

Reply:
The YAML syntax / `hack/verify-kep-metadata.sh` does not allow multiple `alpha:` fields there. It's strange, because multiple feature gates are allowed below in `feature-gates`, but they can't graduate in a different release ?!

I added some comments around.